modules/apps/asset/asset-info-impl/src/main/java/com/liferay/asset/info/internal/item/provider/AssetEntryInfoItemFieldSetProviderImpl.java

```diff
@@ -62,9 +101,49 @@ public InfoFieldSet getInfoFieldSet(String itemClassName) {
 			_infoItemFieldReaderFieldSetProvider.getInfoFieldSet(
 				AssetEntry.class.getName()));
 
+		try {
+			Group group = _groupLocalService.getGroup(scopeGroupId);
+
+			Company company = _companyLocalService.getCompany(
+				group.getCompanyId());
+
+			List<AssetVocabulary> assetVocabularies =
+				_assetVocabularyLocalService.getGroupsVocabularies(
+					new long[] {scopeGroupId, company.getGroupId()},
+					itemClassName);
```

Comment:
It doesn't have to be done for this pull, but it would be a good idea to extract this to a method that is used by both the taglib and this class, which takes a single scopeGroupId and is responsible for adding the companyGroupId in addition. That way, if we later add more things such as inheritance from Asset Libraries or parent sites, then we only have to modify in one place and there is no risk of leaving some cases behind.